salt/modules/virt.py

```diff
@@ -1238,6 +1268,10 @@ def _disk_profile(conn, profile, hypervisor, disks, vm_name):
     pool_caps = _pool_capabilities(conn)
 
     for disk in disklist:
+        # Set default model for cdrom devices before the overlay sets the wrong one
+        if disk.get("device", "disk") == "cdrom" and "model" not in disk:
```

Comment:
Any reason to use `"disk"` here? Or did you mean to use `setdefault`?

Reply:
`"disk"` is the default value if the device attribute doesn't exist... it has nothing to do with the variable name... just bad luck

Reply:
Fair enough - do you think it's worth adding something like this to the comment?

```
# from https://libvirt.org/formatdomain.html#elementsDisks, "device" default is "disk", not None.
```

Reply:
well, not really because this particular piece of code is just ensuring we don't default "cdrom" devices to "virtio" (or another paravirtualized driver). If we start commenting that here, we would need to add it in a number of other places.